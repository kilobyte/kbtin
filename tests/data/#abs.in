#ali m {#math x $0;#show $0 â†’ $x}
m #abs 1.331
m #abs 0
m #abs -0.0001
m #abs -1.331
