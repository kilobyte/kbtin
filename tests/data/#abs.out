#abs 1.331 → 1.331
#abs 0 → 0
#abs -0.0001 → 0.0001
#abs -1.331 → 1.331
